docs/quickstart-java.md

```diff
@@ -0,0 +1,151 @@
+---
+title: Getting Started in Java
+headline: Documentation
+bodyclass: docs
+layout: docs
+---
+
+[//]: <> (The base path for code samples is `_samples/examples/src/main/`.)
+[//]: <> (Change it to `_examples/hello/`.)
+
+<?code-excerpt path-base="../../../../_examples/hello/"?>
```

Comment:
Seems like we aren't using code excepts in this file. Why change the `path-base`?

Reply:
I am going to use it later.